docs/static/schema.json

```
@@ -370,7 +370,7 @@
         },
         "interval": {
           "description": "Sets a different watch interval when using `--watch`, the default being 5 seconds. This string should be a valid Go duration: https://pkg.go.dev/time#ParseDuration.",
-          "$ref": "#/definitions/3/run"
+          "type": "string"
```

@andreynering We could also do something like this so that it validates the syntax of the string. Do we need to support anything other than `ms` and `s`?

```suggestion
          "type": "string"
          "pattern": "^[0-9]+m?s$"
```

---

Although minute (`m`) is unlikely to be used, I'd add it just to be safe.